jedi/evaluate/compiled/__init__.py

```diff
@@ -332,13 +328,13 @@ def dotted_from_fs_path(fs_path, sys_path=None):
     return _path_re.sub('', fs_path[len(path):].lstrip(os.path.sep)).replace(os.path.sep, '.')
 
 
-def load_module(path, name):
+def load_module(path, name, evaluator):
+    sys_path = evaluator.sys_path
```

Comment:
I'm actually worried now that this line might cause some issues. It should probably be a `sys_path_with_modifications` call, but that's not your problem (or at least not a problem of this PR).

I just realized this, but even if it causes issues, there's not a big implication for all normal use cases, so don't care about it too much. It's much more that I should remember this.